packages/env/lib/commands/start.js

```diff
@@ -85,6 +85,15 @@ module.exports = async function start( { spinner, debug } ) {
 					config.coreSource.path,
 					config.coreSource.testsPath
 				);
+
+				// Ensure the tests uploads folder is writeable for travis,
+				// creating the folder if necessary.
+				const testsUploadsPath = path.join(
```

Comment:
Should we chmod more broadly to allow other things to have write access (e.g. chmod all of `wp-content`)? See some discussion around https://github.com/WordPress/gutenberg/pull/22454#issuecomment-631244208 in which a PHP test doesn't have access to what it wants

This may also remove the need for a different `chmod` I added in .travis.yml for the PHP test steps

Reply:
That seems like a good call. I can put together a separate PR to cover it.